package/windows/run.ps1

```diff
@@ -152,6 +147,22 @@ public static class SkipServerCertificateValidation
     [SkipServerCertificateValidation]::Enable()
 } catch {}
 
+# check rancher-agent running #
+$svcAgent = Get-Service -Name "rancher-agent" -ErrorAction Ignore
+if ($svcAgent -and ($svcAgent.Status -eq "Running")) {
+    throw "rancher agent is already on running"
+}
+
+# check msiscsi servcie running #
+$svcMsiscsi = Get-Service -Name "msiscsi" -ErrorAction Ignore
+if ($svcMsiscsi -and ($svcMsiscsi.Status -ne "Running")) {
+    Set-Service -Name "msiscsi" -StartupType Automatic
+    Start-Service -Name "msiscsi" -ErrorAction Ignore
+    if (-not $?) {
+        warn "Start msiscsi service failed"
```

Comment:
Should we throw error here? The our flex volume function won't work without this service.
Or we should program that rancher-agent service depend on msiscsi service

Reply:
iSCSI is just one way to store support, we still support other ways (like SMB, cloud provided storage), so I think we could not throw an error to stop the running action.

how about changing the warning message is enough, like `warn "Start msiscsi service failed, you may not be able to use the iSCSI flexvolume properly."`?